lms/services/digest.py

```diff
@@ -87,6 +88,17 @@ def send_instructor_email_digests(  # pylint:disable=too-many-arguments
             )
 
 
+@dataclass(frozen=True, order=True)
+class AssignmentInfo:
+    """Information about an assignment."""
+
+    id: int
+    tool_consumer_instance_guid: str
+    resource_link_id: str
+    title: str
+    authority_provided_id: str
```

Comment:
The `order=True` just makes `AssignmentInfo`'s sortable. The sort order is arbitrary but it's useful for tests.

I sent a follow-up PR (https://github.com/hypothesis/lms/pull/5768) to rename the existing `UnifiedCourse` and `UnifiedUser` classes to `CourseInfo` and `UserInfo` as I think this is a better naming convention for these local dataclasses.